### PR TITLE
Added C++ resources to help the community flourish

### PR DESCRIPTION
**Pull request by rohitgarg025**
Added C++ resources to help the community flourish

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,6 +146,12 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [C Programming for Beginners](https://youtube.com/playlist?list=PLBlnK6fEyqRggZZgYpPMUxdY1CYkZtARR)
 - [Learn C Org](https://www.learn-c.org)
 
+## C++
+
+- [LearnCPP](https://www.learncpp.com/)
+- [Derek Banas Playlist](https://www.youtube.com/watch?v=DamuE8TM3xo&list=PLGLfVvz_LVvQ9S8YSV0iDsuEU8v11yP9M)
+- [GFG C++ List](https://www.geeksforgeeks.org/c-plus-plus/)
+- [Reference for C++](https://www.cplusplus.com/reference/)
```
Comment (shahednasser) [inline README.md:154]: There's already a C++ section, however, it's misplaced after Express. Please move the resources to that section

File: README.md [hunk 2]
```diff
@@ -146,6 +146,12 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [C Programming for Beginners](https://youtube.com/playlist?list=PLBlnK6fEyqRggZZgYpPMUxdY1CYkZtARR)
 - [Learn C Org](https://www.learn-c.org)
 
+## C++
+
+- [LearnCPP](https://www.learncpp.com/)
+- [Derek Banas Playlist](https://www.youtube.com/watch?v=DamuE8TM3xo&list=PLGLfVvz_LVvQ9S8YSV0iDsuEU8v11yP9M)
+- [GFG C++ List](https://www.geeksforgeeks.org/c-plus-plus/)
```
Comment (shahednasser) [inline README.md:153]: This link is a duplicate so please remove it.

File: README.md [hunk 3]
```diff
@@ -146,6 +146,12 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [C Programming for Beginners](https://youtube.com/playlist?list=PLBlnK6fEyqRggZZgYpPMUxdY1CYkZtARR)
 - [Learn C Org](https://www.learn-c.org)
 
+## C++
+
+- [LearnCPP](https://www.learncpp.com/)
+- [Derek Banas Playlist](https://www.youtube.com/watch?v=DamuE8TM3xo&list=PLGLfVvz_LVvQ9S8YSV0iDsuEU8v11yP9M)
```
Comment (shahednasser) [inline README.md:152]: Change the link to the link of the playlist and make sure the title matches the name of the playlist

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you for contributing @rohitgarg025 I have a few comments below.

**Comment by rohitgarg025** [comment]
@shahednasser  please check my latest commit now. I have incorporated the desired changes. Thanks a lot for guiding.

**Comment by shahednasser** [comment]
Great thanks!

**Pull request merged**